sign_stage_rpms

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+. settings
+
+echo "./list_unsigned_rpms $1 $GPGKEY"
+UNSIGNED_RPMS=$(./list_unsigned_rpms "$1" "$GPGKEY")
+
+for rpm_path in "${UNSIGNED_RPMS[@]}"
+do
+  echo "Signing $rpm_path"
+  ./sign_rpms "$rpm_path"
+done
```

Comment:
(I hate arrays in bash, never grok them).

This calls `./sign_rpms` once per  unsigned rpm, right? I am asking because `./sign_rpms` will interrogate `gopass` about the password for the key on each invocation and also call `rpm` which can add quite some overhead.

I *think* `./sign_rpms` should be able to handle multiple rpms just fine, so we could either `./sign_rpms $(./list_unsigned_rpms)` or (given that list can get a tad long: `echo $UNSIGNED_RPMS | xargs ./sign_rpms`?

Reply:
I'd just use `./list_unsigned_rpms | xargs ./sign_rpms` and avoid `$UNSIGNED_RPMS` altogether. If you want it to print some debug info, make `sign_rpms` echo that it's signing something.